dizoo/league_demo/league_scheduler.py

```diff
@@ -0,0 +1,172 @@
+
+class Scheduler(object):
+    """
+    Overview:
+        Update learning parameters when the trueskill metrics has stopped improving. 
+        For example, models often benefits from reducing entropy weight once the learning process stagnates.
+        This scheduler reads a metrics quantity and if no improvement is seen for a 'patience' number of epochs,
+        the corresponding parameter is increased or decreased, which decides on the 'schedule_mode'.
+    
+    Args:
+        schedule_parameter (str) : One of 'entropy_weight', 'learning_rate'. 
+            Default: 'entropy_weight'
+        schedule_mode (str) : One of 'reduce', 'add','multi','div'. The schecule_mode 
+            decides the way of updating the parameters.  Default:'reduce'.
+        factor (float) : Amount (greater than 0) by which the parameter will be 
+            increased/decreased. Default: 0.05
+        change_range (list) : Indicates the minimum and maximum value 
+            the parameter can reach respectively. Default: [-1,1]
+        threshold (float): Threshold for measuring the new optimum,
+            to only focus on significant changes. Default:  1e-4.
+        optimize_mode (str): One of 'min', 'max', which indicates the sign of 
+            optimization objective. Dynamic_threshold = last_metrics + threshold in `max` 
+            mode or last_metrics - threshold in `min` mode. Default: 'min'
+        patience (int) : Number of epochs with no improvement after which 
+            the parameter will be updated. For example, if `patience = 2`, then we 
+            will ignore the first 2 epochs with no improvement, and will only update 
+            the parameter after the 3rd epoch if the metrics still hasn't improved then. 
+            Default: 10.
+        cooldown (int) : Number of epochs to wait before resuming
+            normal operation after the parameter has been updated. Default: 0.
+    """
+
+    def __init__(self, cfg,
+                # schedule_parameter = 'entropy_weight',
```

Comment:
For the specific config structure and default value of each field, you can follow the implementation of [link](https://github.com/opendilab/DI-engine/blob/main/ding/worker/collector/base_serial_evaluator.py#L36), and add comments about data type, usage, default value and range.

Reply:
fixed.